tsconfig.json

```diff
@@ -0,0 +1,105 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
+
+    /* Projects */
```

Comment:
**question:** Long term, do we need all these commented out things? From the perspective of a future maintainer, it's not clear to me if these are temporarily omitted things that used to be there or just helpful documentation - especially since they have values associated to them. Should we delete the commented out bits now and rely on the doc link for more info?

Reply:
Probably not.  I ended up removing these in #528.  I'll re-add the link to https://aka.ms/tsconfig in that PR.